### ISSUE TITLE
Adjust CONTRIBUTING.md to make it more appropriate for a new lesson

### ISSUE THREAD

**Issue by tobyhodges** [carpentries/workbench-template-md#10]
The template `CONTRIBUTING.md` includes some information, guidance, and links that are not really appropriate for a brand new lesson being created from a template. E.g. in _Where to Contribute_  the current version points to the SWC shell-novice lesson & repository. It also links to the lesson-example repository, and that should be updated to point to a Workbench equivalent.

**Comment by zkamvar** [comment]
I wonder if this should also be the case for {sandpaper} as well since it will not generally create already established lessons.

**Comment by tobyhodges** [comment]
Yes, you are probably right. Creating new template text for this is on my list of projects for an R&R day soon.

**Comment by zkamvar** [comment]
This has been partially addressed in https://github.com/carpentries/sandpaper/pull/407 thanks to @jcolomb

**Comment by zkamvar** [comment]
I realise that this is not exactly addressed, so I'm going to reopen it.
